website/docs/intro.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+title: Introduction
+---
+
+### What is odo?
+
+odo is a fast, iterative and straightforward CLI tool for developers who write, build, and deploy applications on Kubernetes.
+
+odo abstracts the complex Kubernetes terminology so that an application developer can focus on writing code in their favourite framework without having to learn Kubernetes.
+
+odo is focused on [inner loop](./intro#what-is-inner-loop-and-outer-loop) development with some tooling that would help users transition to the [outer loop](./intro#what-is-inner-loop-and-outer-loop).
+
+Brendan Burns, the cofounder of Kubernetes, said in the [book Kubernetes Patterns](https://www.redhat.com/cms/managed-files/cm-oreilly-kubernetes-patterns-ebook-f19824-201910-en.pdf):
```

Comment:
Brendan Burns, the ~cofounder~ **_co-founder_** of Kubernetes,

Reply:
Or "Brendan Burns, ~the cofounder~ **a** co-founder of Kubernetes"

Reply:
🤦🏾‍♂️

Reply:
I hope below is fine:

> one of the co-founders of Kubernetes,